docs/website/src/site/index.md

```diff
@@ -51,4 +51,120 @@ eleventyNavigation:
+</g>
+<path d="M23.0427 59.747C24.9032 59.747 26.4115 58.2388 26.4115 56.3782C26.4115 54.5177 24.9032 53.0095 23.0427 53.0095C21.1822 53.0095 19.6739 54.5177 19.6739 56.3782C19.6739 58.2388 21.1822 59.747 23.0427 59.747Z" fill="#1B6137"/>
+<path d="M48.0311 52.3129C49.8916 52.3129 51.3998 50.8046 51.3998 48.9441C51.3998 47.0836 49.8916 45.5753 48.0311 45.5753C46.1705 45.5753 44.6623 47.0836 44.6623 48.9441C44.6623 50.8046 46.1705 52.3129 48.0311 52.3129Z" fill="#1B6137"/>
+<path d="M41.3939 56.7754C41.3939 56.7754 41.2994 59.3984 38.2061 60.3173C35.1128 61.2362 33.6941 59.0639 33.6941 59.0639" stroke="#1B6137" stroke-width="3.94472" stroke-miterlimit="10" stroke-linecap="round"/>
+<path d="M44.0011 118.362C45.8616 118.362 47.3699 116.854 47.3699 114.993C47.3699 113.133 45.8616 111.625 44.0011 111.625C42.1406 111.625 40.6323 113.133 40.6323 114.993C40.6323 116.854 42.1406 118.362 44.0011 118.362Z" fill="#1138AD"/>
+<path d="M68.9895 110.926C70.85 110.926 72.3583 109.418 72.3583 107.557C72.3583 105.697 70.85 104.188 68.9895 104.188C67.129 104.188 65.6207 105.697 65.6207 107.557C65.6207 109.418 67.129 110.926 68.9895 110.926Z" fill="#1138AD"/>
+<path d="M90.9554 36.1695C91.4847 37.9523 90.1683 38.8122 88.3836 39.3435C86.5988 39.8748 85.0266 39.8748 84.4953 38.092C84.3587 37.6649 84.3087 37.2148 84.3482 36.7681C84.3876 36.3214 84.5159 35.8871 84.7253 35.4906C84.9348 35.0941 85.2212 34.7434 85.5679 34.459C85.9146 34.1746 86.3146 33.9622 86.7444 33.8343C87.1742 33.7064 87.6251 33.6655 88.0709 33.7141C88.5167 33.7627 88.9483 33.8997 89.3405 34.1172C89.7326 34.3347 90.0775 34.6282 90.3548 34.9806C90.6321 35.333 90.8363 35.7372 90.9554 36.1695Z" fill="#992224"/>
+<path d="M115.942 28.7334C116.473 30.5181 115.155 31.3761 113.37 31.9073C111.585 32.4386 110.013 32.4386 109.482 30.6559C109.345 30.2288 109.295 29.7787 109.335 29.332C109.374 28.8853 109.502 28.451 109.712 28.0545C109.921 27.658 110.208 27.3073 110.554 27.0229C110.901 26.7385 111.301 26.5261 111.731 26.3982C112.161 26.2703 112.612 26.2294 113.057 26.278C113.503 26.3266 113.935 26.4636 114.327 26.6811C114.719 26.8985 115.064 27.1921 115.341 27.5445C115.618 27.8969 115.823 28.3011 115.942 28.7334Z" fill="#992224"/>
+<path class="ds-explore__components-part" d="M118.209 105.558L121.568 111.638" stroke="#5985FF" stroke-width="5.25962" stroke-miterlimit="10" stroke-linecap="round"/>
+<path class="ds-explore__components-part" d="M125.346 84.7039L131.64 81.7661" stroke="#5985FF" stroke-width="5.25962" stroke-miterlimit="10" stroke-linecap="round"/>
+<path class="ds-explore__components-part" d="M124.954 95.9535L133.071 98.6218" stroke="#5985FF" stroke-width="5.25962" stroke-miterlimit="10" stroke-linecap="round"/>
+</svg>
+
+
+</div>
+
+<div class="ds-heading-2">
+<a href="/categories/components/overview/" aria-label="Components. Sanoma Learning’s UI components are built to best practices to ensure usability, performance and accessibility.">
```

Comment:
should the label be the same as the link text? doesn't that read the same text aloud twice?

Reply:
when i use the arrowkeys to navigate over all the elements (and don't click the link) i get the same text twice; once from the aria-label and once from the paragraph below it